test/src/System/Time.cpp

```diff
@@ -0,0 +1,172 @@
+#include <SFML/System/Time.hpp>
+
+#include <catch2/catch.hpp>
```

Comment:
Guess this only applies, once #1898 is merged 🙂

Reply:
My bad. Should be fixed now.